frontend/src/components/teacher/view-sessions/TestSessionListItem.tsx

```diff
@@ -23,8 +23,13 @@ import TestSessionListItemPopover from "./TestSessionListItemPopover";
 
 export type TestSessionListItemProps = {
   testSessionId: string;
-  classroomName?: string;
+  testId: string;
   testName: string;
+  classroomId: string;
+  classroomName: string;
+  endDate: Date;
+  startDate: Date;
+  notes?: string;
```

Comment:
There are enough props here that I'm wondering if we should just pass down `session` using the response type directly. That way we don't have to enumerate all of these props. Thoughts?